spec/util/java_version_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+require 'logstash/util/java_version'
+
+describe "LogStash::Util::JavaVersion" do
+  subject(:mod) { LogStash::Util::JavaVersion }
+
+  it "should get the current java version if we're on Java" do
+    if LogStash::Environment.jruby?
+      expect(LogStash::Util::JavaVersion.version).to be_a(String)
+    end
+  end
+
+  it "should mark a bad beta version as bad" do
+    expect(mod.bad_java_version?("1.7.0_45-beta")).to be_truthy
```

Comment:
What happen if I run the test without being java? you should skip the test if not under jruby?, do something like: https://github.com/elastic/logstash/blob/master/spec/util/json_spec.rb#L33